scripts/collections.js

```diff
@@ -148,46 +204,80 @@ export async function createCollection(title, description, items) {
 }
 
 /**
- * Lists collections with optional limit and cursor parameters.
+ * Search Lists collections with optional limit and page parameters.
  *
  * @param {number} - The optional maximum number of collections to retrieve.
- * @param {string} - The optional cursor for paginating the results.
+ * @param {number} - The optional page number for paginating the results.
  * @returns {Promise<object>} A promise that resolves with a list of collections.
  * @throws {Error} If an HTTP error or network error occurs.
  */
-export async function listCollection(limit = undefined, cursor = '') {
+
+export async function searchListCollection(limit = undefined, page = 0) {
   // Construct the query parameters
   const queryParams = new URLSearchParams();
 
   if (limit) {
     queryParams.append('limit', limit);
   }
 
-  if (cursor) {
-    queryParams.append('cursor', cursor);
+  if (page) {
+    queryParams.append('page', page);
   }
 
+  const adminConfig = await getAdminConfig();
+  const indexName = adminConfig.searchCollectionIndex;
```

Comment:
Why are we taking collection index from config. It is {env}_collections. Example 12345-689_collections.

Reply:
Just to be on safer side we made this configurable, in case something changes in future